run_unittests.py

```diff
@@ -883,10 +884,14 @@ def test_compiler_detection(self):
     def test_always_prefer_c_compiler_for_asm(self):
         testdir = os.path.join(self.common_test_dir, '141 c cpp and asm')
         # Skip if building with MSVC
-        env = Environment(testdir, self.builddir, self.meson_command,
-                          get_fake_options(self.prefix), [])
-        if env.detect_c_compiler(False).get_id() == 'msvc':
-            raise unittest.SkipTest('MSVC can\'t compile assembly')
+        #env = Environment(testdir, self.builddir, self.meson_command,
+        #                  get_fake_options(self.prefix), [])
+        #if env.detect_c_compiler(False).get_id() == 'msvc':
```

Comment:
__[flake8]__
* [E265] block comment should start with '# '

<!-- Comment by [SideCI](https://sideci.com) -->